app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -589,7 +589,8 @@ class LuciBuildService {
             target: target,
             task: task,
             priority: kRerunPriority,
-            properties: commit.slug == Config.engineSlug ? Config.engineDefaultProperties : null,
+            properties:
+                commit.slug == Config.engineSlug || commit.slug == Config.flutterSlug ? Config.defaultProperties : null,
```

Comment:
Why not make this a default property for all the repos?

Reply:
Updated, no reason for not being the default.